src/app/slash_plugins/gamble.plugin.ts

```diff
@@ -0,0 +1,98 @@
+import { MessageEmbed } from 'discord.js';
+import { ISlashCommand } from '../../common/slash';
+import { IContainer } from '../../common/types';
+
+const MIN_BET: number = 10;
+
+const plugin: ISlashCommand = {
+  name: 'gamble',
+  commandName: 'gamble',
+  description: 'Bet your Tacos on a coin flip',
+  options: [
+    {
+      name: 'amount',
+      description: 'The amount of Tacos to gamble',
+      type: 'STRING',
+      required: true,
+    },
+  ],
+  execute: async ({ interaction, container }) => {
+    const arg = interaction.options.getString('amount');
+    if (arg === null || !validateBetAmount(arg)) {
+      await interaction.reply('Invalid bet amount.');
+      return;
+    }
+
+    await interaction.deferReply();
+
+    const userDoc = await container.pointService.getUserPointDoc(interaction.user.id);
+    const pointsToGamble = arg.toLowerCase() === 'all' ? userDoc.numPoints : parseInt(arg);
+
+    if (userDoc.numPoints === 0) {
+      await interaction.followUp('You have no Tacos to gamble!');
```

Comment:
```suggestion
      await interaction.followUp({ content: 'You have no Tacos to gamble!', ephemeral: true });
```

Reply:
I need to remember that is an option lol